iceberg/iceberg-handler/src/test/results/positive/delete_iceberg_copy_on_write_partitioned.q.out

```diff
@@ -121,140 +121,140 @@ STAGE PLANS:
             Map Operator Tree:
                 TableScan
                   alias: tbl_ice
-                  Statistics: Num rows: 6 Data size: 582 Basic stats: COMPLETE Column stats: COMPLETE
+                  Statistics: Num rows: 6 Data size: 588 Basic stats: COMPLETE Column stats: PARTIAL
```

Comment:
What is the reason of the column stats loss?

Reply:
after the update/delete Hive doesn't recompute stats and on the next insert it only writes stats generated by that insert since prev stats is not accurate/no stats for prev snapshot. However, we had a bug and  did a fallback to HMS and used whatever was previously stored there (no snapshot id validation, similar to ACID writeIds)